#
# Command signal
#

# Propeller speed
float64 n

# Angle of the lateral rudder
float64 delta_r

# Angle of the elevation rudder
float64 delta_e

# Rate of the ballast water pump
float64 P
